### PR TITLE
Re add rebecca

### PR DESCRIPTION
**Pull request by beccastevens**

### CODE REVIEW

**Review by Hellmark — APPROVED** [review]
WOOHOO!

File: content/code-of-conduct.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ list below:
 - Becca Stevens (@beccastevens)
 - Keith Brown (@hellmark)
 - Craig Buchek (@craigbuchek)
+- Rebecca Skinner (@rebecca.skinner)
 - Steven Pritchard (@silug)
+>>>>>>> master
```
Comment (booch) [inline content/code-of-conduct.md]: Err, you seem to have merge garbage in both of these files.

**Review by booch — CHANGES_REQUESTED** [review]
Need to get rid of the non-Markdown merge conflict indicators.

File: content/code-of-conduct.md (hunk 1 above)
Comment (beccastevens) [reply content/code-of-conduct.md]: derp, good catch thanks

**Review by booch — APPROVED** [review]

**Review by silug — APPROVED** [review]

**Pull request merged**